MaterialLib/MPL/Properties/CreatePermeabilityOrthotropicPowerLaw.cpp

```diff
@@ -0,0 +1,79 @@
+/**
+ * \file
+ * \copyright
+ * Copyright (c) 2012-2020, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ */
+
+#include "BaseLib/ConfigTree.h"
+#include "ParameterLib/CoordinateSystem.h"
+#include "PermeabilityOrthotropicPowerLaw.h"
+
+namespace MaterialPropertyLib
+{
+std::unique_ptr<Property> createPermeabilityOrthotropicPowerLaw(
+    BaseLib::ConfigTree const& config,
+    ParameterLib::CoordinateSystem const* const local_coordinate_system)
+{
+    //! \ogs_file_param{properties__property__type}
+    config.checkConfigParameter("type", "PermeabilityOrthotropicPowerLaw");
+    DBUG("Create PermeabilityOrthotropicPowerLaw solid phase property");
+
+    auto const intrinsic_permeabilities =
+        //! \ogs_file_param{properties__property__PermeabilityOrthotropicPowerLaw__intrinsic_permeabilities}
+        config.getConfigParameter<std::vector<double>>(
+            "intrinsic_permeabilities");
+    /*
```

Comment:
Can this be removed?

Reply:
I, actually, have included the tests. in the rebased commits... Somehow the commented sections slipped past my QC ;)